scripts/e2e_check_pod_restarts.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
/usr/bin/env for portability, and consistency with other scripts.

Reply:
Done